utopia-remix/app/components/sharingDialog.tsx

```diff
@@ -59,26 +60,40 @@ function SharingDialog({ project }: { project: ProjectListing | null }) {
   const setSharingProjectId = useProjectsStore((store) => store.setSharingProjectId)
   const accessRequests = useProjectsStore((store) => store.sharingProjectAccessRequests)
 
+  // keep track of possible new access levels to avoid jumpy single-frame updates
+  const [currentAccessLevel, setCurrentAccessLevel] = React.useState<AccessLevel | null>(null)
```

Comment:
Can we set to the real access level immediately? And then we don't need the nullability and the null checks.